src/lib/Menu/ContentRightSidebarBuilder.php

```diff
@@ -193,6 +216,94 @@ public function createStructure(array $options): ItemInterface
             ),
         ]);
 
+        $currentUser = $this->userService->loadUser(
+            $this->permissionResolver->getCurrentUserReference()->getUserId()
+        );
+        /** @var \Ibexa\Contracts\Core\Repository\Values\Content\Field $currentUserAccount */
+        $currentUserAccount = $currentUser->getField('user_account');
+        /** @var string $currentUserLanguageCode */
+        $currentUserLanguageCode = $currentUserAccount->getLanguageCode();
```

Comment:
tbh, I do not get this part, why previews options are dependent on user content language? I would say that maybe language user settings make more sense here.

@tischsoic